src/forms/form.js

```diff
@@ -5,21 +5,23 @@ import isEmpty from 'lodash/isEmpty';
 import { stringMapper } from './mappers';
 import { noValidation } from './validators';
 
+export type RawValue = string | Array<any>;
```

Comment:
Can you add a test where your using the array part? I don't see how the array part solves or problem without an example.